_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,83 @@
+
+## Create an agenda for events and fairs
+
+Throughout the year ePages is organizing internal events like our [hackathon](/blog/events/this-years-hackathon-a-travel-through-time/){:target="_blank"}, our Sales Kick-off event, or the [ePages YOU](/blog/events/epages-you-2019-from-a-perspective-of-two-newbies/){:target="_blank"}.
+But ePages is also participating in external events such as conferences and fairs.
+These events can also provide perfect content for your company Instagram channel.
+In order to keep them in mind create an agenda where you list all internal and external events.
+Note also which team or employees do visit the external events.
+Thus, you can remind them on time to take some nice pictures of the event.
+
+## Create a content plan
+
+In order to coordinate your content on Instagram you can create a content plan.
+This is a calender where you can note which picture do you want to share when.
```

Comment:
I'd say it like this: This content plan can be understood as a calendar in which you can note down which picture you'd like to share on which day.